### PR TITLE
Make Create Contest title gray to match Create Course title

### PR DESCRIPTION
**Pull request by sudarsan2k5**
# Description

Changed the New Contest header styling from blue to light gray with dark text for better visual consistency.

image

Fixes: #7893

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.

### CODE REVIEW

**Comment by sudarsan2k5** [comment]
@heduenas I directly added the style to meet the requirement. I don’t think it will cause any issues, but let me know if you have any suggestions.

File: frontend/www/js/omegaup/components/contest/NewForm.vue [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <template>
   <div class="card">
-    <div v-if="!update" class="card-header bg-primary text-white panel-heading">
+    <div
+      v-if="!update"
+      class="card-header"
+      style="background-color: rgba(0, 0, 0, 0.03); color: #212529"
+    >
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/contest/NewForm.vue:7]: In general, we avoid using inline styles due to the potential risk of [cross-site scripting](https://chatableapps.com/technology/securing-your-website-exploring-the-dangers-of-unsafe-inline-style-src-and-how-to-safeguard-against-it/) attacks.

Therefore, our first option is always to use the classes provided by [Bootstrap](https://getbootstrap.com/docs/4.0/getting-started/introduction/).

If no class satisfies the style we need, our second option is to create our own styles using classes within our components.

For the case you are working on, you can refer to the example in the "New Course" component to replicate the same classes, which might be sufficient.

**Review by pabo99 — COMMENTED** [review]
Just one comment, and for me it's okay

File: frontend/www/js/omegaup/components/contest/NewForm.vue (hunk 1 above)
Comment (sudarsan2k5) [reply frontend/www/js/omegaup/components/contest/NewForm.vue:7]: Got it! Thank you 😊.

**Comment by sudarsan2k5** [comment]
I changed the styling using the Bootstrap class, and it's working fine now. Let me know if any further modifications are needed.

**Review by pabo99 — APPROVED** [review]
Excellent! Thank you @sudarsan2k5 !!

**Review by heduenas — APPROVED** [review]

**Pull request merged**